js/main.js

```diff
@@ -0,0 +1,15 @@
+const mainMenu = document.querySelector('.mainMenu');
+const openMenu = document.querySelector('.openMenu');
+const closeMenu = document.querySelector('.closeMenu');
+
+function openMobileMenu() {
+  mainMenu.style.display = 'flex';
+  mainMenu.style.top = '0';
+}
+
+function closeMobileMenu() {
+  mainMenu.style.top = '-100%';
+}
+
+openMenu.addEventListener('click', openMobileMenu);
+closeMenu.addEventListener('click', closeMobileMenu);
```

Comment:
### To match the Figma design:
- [ ] Kindly add the blur effect for the background of the menu.
- [ ] Kindly change the `font-weight` for the menu item.
- [ ] Kindly increase the right spacing a little bit.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/161545571-84e40e80-0693-4e56-bc56-a712a8d19fc3.png) | ![image](https://user-images.githubusercontent.com/24703171/161545794-204de2da-a871-4195-8dc7-b4b064aa7adc.png) |